src/lib/Navbar.svelte

```diff
@@ -0,0 +1,131 @@
+<script lang="ts">
+	import { page as currentPage } from '$app/stores';
+	import { mdiMenu, mdiClose } from '@mdi/js';
+
+	export let logoUrl: string;
+	export let siteName: string;
+	type Page = { name: string; url: string };
+	export let pages: readonly Page[];
+
+	let menuExpanded = false;
+</script>
+
+<header>
+	<div class="container">
+		<nav>
+			<ul>
+				<li>
+					<a href="/" class="brand contrast"><img src={logoUrl} alt="" />{siteName}</a>
```

Comment:
Clicking this should also close the menu

Reply:
fixed in [5a1c5f0](https://github.com/kendoclub-at-umich/website/pull/40/commits/5a1c5f06d6191bba80372956b4c8541e5264eb5a)

Reply:
Also thanks for giving me the chance to figure out how to do that one on my own. I kind of panicked at first, but calmed myself down and realized I had everything I needed to figure this out bc I understand the code.

Reply:
I knew you could figure it out. You got close but your code also opens the menu when you click the brand link. It should probably only close it.

Reply:
Oooooops 😂
Fixed in [faa773e](https://github.com/kendoclub-at-umich/website/pull/40/commits/faa773ecdb5912c82253d218f2a7559880e5695e)